translations/Hindi/special-terms.md

```diff
@@ -0,0 +1,179 @@
+
+# Terms
+
+* Message: संदेश
+
+  Message has a direct translation in hindi
+
+  टिम(Tim) ने आप एक प्राइवेट संदेश भेजा है।
+* Private Message (PM): निजी संदेश
+
+  निजी is the direct translation for Private in Hindi.
+
+  टिम(Tim) ने आप एक निजी संदेश भेजा है।
+* Group PM: समूह निजी संदेश
+
+  Group and message have direct translations in Hindi.
+
+  टिम(Tim) ने आप और राम के लिए एक समूह निजी संदेश भेजा है।
+
+* Realm: संगठन / टीम
+
+  Realm has a word-to-word translation of क्षेत्र in Hindi, but क्षेत्र
+  directly translates to Province in English which is clearly inappropriate for
+  Zulip as Realm in Zulip documents is interpreted as “Organization”.
+  Therefore, I propose the use of Organization or Team(direct word translation
+  in Hindi) instead. IMO using Team will be more appropriate.
+
+  आप ने एक संगठन/टीम बनाया है।
+
+* Stream: धारा
+
+  Direct translation of the word "Stream". I thought of different translations
+  like चैनल but that doesn't fit will with the Zulip context.
+
+  आप को बैकएंड धारा में सूचनाओं की है।
+
+* Topic: विषय
+
+  This is a direct translation and fits perfectly.
+
+  आप एक विषय के बिना कोई धारा पे संदेश नहीं भेज सकते हैं।
+
+* Private/Invite-Only Stream: प्राइवेट धारा
+
+  निजी is a direct translation for Private and fits well in Zulip's context,
+  therefore प्राइवेट धारा seems apt for this.
+
+  आप को एक निजी धारा मे आमंत्रित किया गया है ।
```

Comment:
It is not clear that are you proposing प्राइवेट धारा or निजी धारा 
You are using निजी धारा in the sentence here and प्राइवेट धारा in explanation.